README.md

```diff
@@ -131,6 +131,17 @@ func main() {
 
 </details>
 
+### Deprecation and Sunset
+
+If the Equinix Metal API returns a [RFC-8594](https://tools.ietf.org/html/rfc8594) `Deprecation` or `Sunset` header, packngo will log this header to stderr with any accompanied `Link` headers.
```

Comment:
This should also mention the deprecation draft spec: 
https://tools.ietf.org/html/draft-ietf-httpapi-deprecation-header-01#section-2